sentry-android-core/src/main/java/io/sentry/android/core/LifecycleWatcher.java

```diff
@@ -122,6 +122,7 @@ public void run() {
                   scopes.endSession();
                 }
                 scopes.getOptions().getReplayController().stop();
+                scopes.getOptions().getContinuousProfiler().stopAllProfiles();
```

Comment:
This means the profiler will continue to run for 30s in the background before stopping, is this what we want?

Reply:
yes, and actually stops after 30s and the last chunk is finished

Reply:
Alright, to be honest I still wouldn't do this. Once the app is in background, any executed code can easily produce a background ANR.

Reply:
true, but if we don't, the profiler could continue to run indefinitely, right?